.github/workflows/rust.yml

```diff
@@ -8,31 +8,31 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Set up Rust
-        uses: hecrj/setup-rust-action@v1
+        uses: hecrj/setup-rust-action@v2
```

Comment:
I am a little worried about this action as it doesn't look very actively maintained 

https://github.com/hecrj/setup-rust-action/issues

However, this is probably ok -- maybe we can find time to setup rust ourselves (using `rustup install`)

Here is how we do it in DataFusion: https://github.com/apache/datafusion/blob/main/.github/actions/setup-builder/action.yaml

Reply:
@alamb Thanks for your review and comment. I will be happy to do this when I get time.